llvm/include/llvm/CAS/CASNodeSchema.h

```diff
@@ -0,0 +1,72 @@
+//===- llvm/CAS/CASNodeSchema.h ---------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CAS_CASNODESCHEMA_H
+#define LLVM_CAS_CASNODESCHEMA_H
+
+#include "llvm/CAS/CASReference.h"
+#include "llvm/CASObjectFormats/Data.h"
+#include "llvm/ExecutionEngine/JITLink/JITLink.h"
+#include "llvm/Support/ExtensibleRTTI.h"
+
+namespace llvm {
+namespace cas {
+
+/// NodeSchema for object hierarchies.
+///
```

Comment:
Bonus comment line

Reply:
Fixed in 3c070754a30423ec21c4bb2b0f3366756c03600c.